### PR TITLE
[CKLS-20652] Remove the twig template error 

### PR DESCRIPTION
**Pull request by Alex-Caepio**

### CODE REVIEW

**Review by ayakhnenka-ck — APPROVED** [review]

File: Controller/PanelController.php [hunk 1]
```diff
@@ -9,34 +9,43 @@
...
+        $this->twig = $twig;
+    }
+
     /**
      * This method renders the global Propel configuration.
      */
     public function configurationAction()
     {
-        $templating = $this->container->get('templating');
+        $configuration = $this->container->get('propel.configuration')->getParameters();
```
Comment (fchastanet) [inline Controller/PanelController.php:39]: we could inject propel.configuration into the constructor too

File: Resources/views/Collector/propel.html.twig [hunk 2]
```diff
@@ -73,33 +73,33 @@
...
-
-                        {% if app.request.query.get('query', -1) != i %}
-                            - <a href="{{ path('_profiler', {'panel': 'propel', 'token': token, 'connection': query.connection, 'query': i}) }}#propel-query-{{ i }}">Explain the query</a>
+                <tr>
+                    <td>
+                        <a name="propel-query-{{ i }}" ></a>
+                        <code>{{ query.sql|format_sql }}</code>
+                        {% if app.request.query.has('query') and app.request.query.get('query') == i %}
+                            <div class="SQLExplain">
+                                {{ render(controller('@Propel/Panel/explain', {
```
Comment (seb-fabre) [inline Resources/views/Collector/propel.html.twig]: Isn't it inconsistent with line 104? Here we replace the syntax to use slashes, but for the same render(controller(...)) in line 104 we keep the syntax with the colons, is there a reason?

Comment (Alex-Caepio) [reply Resources/views/Collector/propel.html.twig]: Alex had the same question. But I saw that changing 104 line by using @ caused the error in the logs. It was saying that the controller is not found. I can try to change the 82 line back to use :.

Comment (Alex-Caepio) [reply Resources/views/Collector/propel.html.twig]: I changed the 82 line to PropelBundle:Panel:explain and the Symfony profiler seems to be working fine.

**Review by seb-fabre — APPROVED** [review]

**Review by seb-fabre — APPROVED** [review]

**Review by fchastanet — APPROVED** [review]

**Pull request merged**